service/src/test/java/com/kodemon/service/test/facade/FightFacadeTest.java

```diff
@@ -295,7 +295,16 @@ private void prepareFights() {
 
         fights.add(fight1);
         fights.add(fight2);
+    }
 
-        when(beanMappingService.mapTo(trainerFights, FightDTO.class)).thenReturn(fights);
+    @AfterMethod
+    void resetMocks() {
+        Mockito.reset(beanMappingService);
```

Comment:
I wrote to Miso's PR, that the `reset()` method accept a vararg, so you could just write `reset(beanMappingService, trainerService, ...)`.